templates/autoscale-new-vpc.template

```diff
@@ -0,0 +1,1163 @@
+{
+    "AWSTemplateFormatVersion": "2010-09-09",
+    "Description": "Fortinet FortiGate Auto Scaling [new VPC]. This template creates a new VPC and deploys Fortinet FortiGate Auto Scaling into it. For more details, please see the Fortinet FortiGate Auto Scaling documentation. **WARNING** You will be billed for the FortiGate instances and related AWS resources if you create a stack from this template.",
+    "Parameters": {
+        "ResourceTagPrefix": {
+            "Type": "String",
+            "AllowedPattern": "[0-9a-zA-Z@.#-]+",
+            "MaxLength": 50,
+            "ConstraintDescription": "Resource tag prefix can include numbers, lowercase letters, uppercase letters, ampersat(@) , hyphens (-), period (.), and hash (#). Max length is 50.",
+            "Description": "The ResourceGroup Tag Key used on all resources and as the name prefix of all applicable resources. Can only contain numbers, lowercase letters, uppercase letters, ampersat(@), hyphens (-), period (.), and hash (#). Max length is 50."
+        },
+        "CustomIdentifier": {
+            "Type": "String",
+            "Default": "fgtASG",
+            "MaxLength": "10",
+            "AllowedPattern": "[A-Za-z0-9]+",
+            "ConstraintDescription": "must only contain uppercase and lowercase letters and numbers",
+            "Description": "An alternative name prefix to be used on a resource that the 'Resource tag prefix' cannot apply to. Can only contain numbers, lowercase letters, and uppercase letters. Max length is 10."
```

Comment:
```suggestion
            "Description": "An alternative name prefix to be used on a resource that the 'Resource tag prefix' cannot apply to. It can only contain numbers, lowercase letters, and uppercase letters. Max length is 10."
```